buildspec/release/70checkmarketplace.yml

```diff
@@ -0,0 +1,48 @@
+version: 0.2
+
+phases:
+    install:
+        runtime-versions:
+            nodejs: 16
+
+        commands:
+            - apt update
+            - apt install -y wget gpg
+            - wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > packages.microsoft.gpg
+            - install -o root -g root -m 644 packages.microsoft.gpg /etc/apt/trusted.gpg.d/
+            - sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main" > /etc/apt/sources.list.d/vscode.list'
```

Comment:
`wget` isn't needed, use `curl` instead. The other steps here could perhaps match up better with our other scripts: https://github.com/aws/aws-toolkit-vscode/blob/74aca739066f0ba6f89d6b812f075f8dd2be7371/buildspec/release/50githubrelease.yml#L12-L14

but that's not a high priority, shouldn't block this PR.